index.bs

```diff
@@ -5444,18 +5447,14 @@ ECMAScript value or failure, or the steps may throw an exception.
       <dd>
         1. If Type(|value|) is not Object, return failure.
 
-        2. Let |hop| be HasOwnProperty(|value|, |identifier|)
-
-        3. Assert: |hop| is not an abrupt completion.
+        2. Let |hop| be [=!=] HasOwnProperty(|value|, |identifier|)
```

Comment:
There's more context in the spec (a note below the algorithm); the value being processed here is always the result of a structured clone operation, so there must be no proxies, getters, etc.